tests/spread/extensions/gnome/task.yaml

```diff
@@ -16,7 +16,6 @@ prepare: |
   . "$TOOLS_DIR/snapcraft-yaml.sh"
   set_base "$SNAP_DIR/snap/snapcraft.yaml"
   sed -i 's/gnome-3-28/gnome/' "$SNAP_DIR/snap/snapcraft.yaml"
-  sed -i 's/bin\/gtk3-hello/usr\/local\/bin\/gtk3-hello/' "$SNAP_DIR/snap/snapcraft.yaml"
```

Comment:
this is not a hack, this most likely means we are changing the expected "command" entry for all snaps

Reply:
When we set `CMAKE_INSTALL_PREFIX` to empty, binaries are installed in `bin/` instead of `usr/local/bin/`. This matches the installation path used in the original v1 plugin for core18.

Reply:
The safest solution in this case is to not set the install prefix in the plugin, but orient developers using base core22 (and core20) to add `-DCMAKE_INSTALL_PREFIX=` to `cmake-parameters` to have core18 plugin compatibility. In this case, we can close this PR and update craft-parts to 1.12.1 rolling back the install prefix change, but keeping the prefix path fix.